src/luascript.cpp

```diff
@@ -9787,6 +9788,26 @@ int LuaScriptInterface::luaPlayerRemoveItem(lua_State* L)
 	return 1;
 }
 
+
+int LuaScriptInterface::luaPlayerSendSupplyUsed(lua_State* L)
+{
+	// player:sendSupplyUsed(item)
+	Player* player = getUserdata<Player>(L, 1);
+	if (!player) {
+		lua_pushnil(L);
+		return 1;
+	}
+
+	Item* item = getUserdata<Item>(L, 2);
+	if (!item) {
```

Comment:
Maybe 
```Item* item = getUserdata<Item>(L, 2);
if (item) {
    player->sendSupplyUsed(item);
} else {
    lua_pushnil(L);
}
return 1;
```

Reply:
This follows the default from luascript.cpp
![image](https://user-images.githubusercontent.com/53269692/158015138-c5d266be-5cd2-4ff2-aef0-aada4d38bf22.png)